woocommerce/woocommerce
# shipping zone settings: fixed title for shipping methods…

…without settings

### All Submissions:


### Changes proposed in this Pull Request:

Fixes shipping method title for methods without settings (omitting the text from the Edit & Delete links that now appear in the title erroneously). Mostly aesthetic fix.

### How to test the changes in this Pull Request:

1. In Woocommerce Settings > Shipping, enter the Edit page for a Shipping Zone.
2. Add a new Shipping method, that doesn't support `settings` or `instance-settings` in its class. 
3. The title for this method should appear as plain text and not as a link. The title shouldn't include any "Edit" or "Delete" texts (these appear as hover action links for methods WITH settings).

### Other information:




### Changelog entry

shipping zone settings: fixed title for shipping methods without settings
